discover.go

```diff
@@ -19,7 +19,11 @@ package openid
 // used as both the Claimed Identifier and the OP-Local Identifier
 // when an OP Identifier is entered.
 func Discover(id string) (opEndpoint, opLocalID, claimedID string, err error) {
-	return discover(id, urlGetter)
+	return discover(id, defaultInstance.urlGetter)
+}
+
+func (oid *OpenID) Discover(id string) (opEndpoint, opLocalID, claimedID string, err error) {
```

Comment:
Funny, that's how I originally wrote it. I don't even remember why I rewrote it the way the pull request is now. Probably for the patch to be smaller. I'll change it back.